src/commands/fun/randommeme.js

```diff
@@ -0,0 +1,29 @@
+const SA = require('superagent')
+const Command = require('../../classes/Command')
+
+module.exports = new Command(async msg => {
+  try {
+    const res = await SA.get(`https://api.imgur.com/3/g/memes/viral/${Math.floor((Math.random() * 8) + 1)}`)
+      .set('Authorization', 'Client-ID ' + process.env.IMGUR_KEY)
+    const ctx = res.body.data[Math.floor((Math.random() * 20) + 1)]
+    await msg.channel.createMessage({
+      embed: {
+        url: ctx.link,
+        title: ctx.title,
+        image: {
+          url: ctx.images[0].link
+        },
+        footer: {
+          text: 'imgur.com',
+          icon_url: 'https://i.imgur.com/JFa7xb3.png'
```

Comment:
Don't imgur links expire after 90 days without an account? This might not be the best option.

Reply:
I uploaded this on my own imgur account so expiry shouldn't be an issue, however I still agree that this is not a great option.